hot/update-panel-elems.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function updatePanelElems(elemName, updateFn) {
+  const elems = document.querySelectorAll(`body /deep/ ${elemName}`);
+  let numUpdated = 0;
+  for (const elem of elems) {
+    if (elem._config && elem.update) {
```

Comment:
Is this intended to detect whether it's a panel component? I'd suggest that checking for the presence of `elem.panelID` is more robust (less prone to collisions and doesn't depend on internal prop).